json.schemastore.org/cargo.json

```diff
@@ -1513,9 +1513,10 @@
     },
     "Authors": {
       "title": "Authors",
-      "description": "The `authors` field lists people or organizations that are considered the\n\"authors\" of the package. The exact meaning is open to interpretation — it may\nlist the original or primary authors, current maintainers, or owners of the\npackage. These names will be listed on the crate's page on\n[crates.io](https://crates.io). An optional email address may be included within angled\nbrackets at the end of each author.\n\n> **Note**: [crates.io](https://crates.io) requires at least one author to be listed.",
+      "description": "The `authors` field lists people or organizations that are considered the\n\"authors\" of the package. An optional email address may be included within angled\nbrackets at the end of each author.\n\n> **Note**: [crates.io](https://crates.io) requires at least one author to be listed.",
       "type": "array",
       "uniqueItems": true,
+      "deprecated": true,
```

Comment:
https://github.com/rust-lang/rfcs/pull/3052

Following this PR, it appears that it has become optional input rather than deprecated.

Reply:
Indeed, as you said.

Many people will be surprised, but we'll merge this.

Please make the corrections pointed out in another comment.